src/v/kafka/protocol/types.h

```diff
@@ -136,4 +136,20 @@ concept KafkaApi = requires(T request) {
     { T::min_flexible } -> std::convertible_to<const api_version&>;
 };
 
+/*
+ * Data type of the configuration entry.
+ */
+enum class describe_configs_type : int8_t {
+    unknown = 0,
+    boolean = 1,
+    string = 2,
+    _int = 3,
```

Comment:
you might consider `int_type` or another convention like `iint` or `intt`. technically c++ doesn't allow `_` prefix. i realize we use it for private members, but we should probably not perpetuate that practice.

Reply:
I'll go with `int_type`